OpenDreamRuntime/Objects/MetaObjects/DreamMetaObjectAtom.cs

```diff
@@ -31,6 +31,12 @@ sealed class DreamMetaObjectAtom : IDreamMetaObject {
             VerbLists[dreamObject] = new VerbsList(_objectTree, dreamObject);
             _filterLists[dreamObject] = new DreamFilterList(_objectTree.List.ObjectDefinition, dreamObject);
 
+            // TODO: These should use their own special list types
+            dreamObject.SetVariable("overlays", new(_objectTree.CreateList()));
+            dreamObject.SetVariable("underlays", new(_objectTree.CreateList()));
+            dreamObject.SetVariableValue("vis_locs", new(_objectTree.CreateList()));
+            dreamObject.SetVariableValue("vis_contents", new(_objectTree.CreateList()));
```

Comment:
But in the DM file these two still do 
```
var/list/vis_locs = list() as opendream_unimplemented
var/list/vis_contents = list() as opendream_unimplemented
```
?

I also don't see where you internally create a list for `locs` or `content`

Reply:
Oops, must have missed that while fixing a merge conflict

`locs` is already created on each access inside DreamMetaObjectMovable

`contents` is different depending on the type of atom, they all have their own getter